pkg/vetter/mtlsprobes/README-mtls-probes-incompatible.md

```diff
@@ -19,12 +19,155 @@ that will be trusted by other sidecars.  However, the liveness probe or
+    spec:
+      containers:
+      - image: docker.io/citizenstig/httpbin
+      imagePullPolicy: IfNotPresent
+      name: httpbin
+      ports:
+        - containerPort: 8000
+      livenessProbe:
+        httpGet:
+          path: /status/200
+          port: 8000
+        initialDelaySeconds: 5
+        periodSeconds: 5
+```
+From here, running `kubectl get pods -l app=httpbin` would produce output
+similar to the following:
+
+```shell
+NAME                       READY     STATUS             RESTARTS   AGE
+httpbin-1a23bc456d-7ef8g   1/2       CrashLoopBackOff   3          1m
+```
+If the Pod is deployed in the mesh with the above configuration, the following
+note will be generated:
+
+```shell
+Summary: "mTLS and liveness probe incompatible - httpbin-1a23bc456d-7ef8g"
+
+Message: "ERROR: The pod httpbin-1a23bc456d-7ef8g in namespace default uses
+liveness probe which is incompatible with mTLS. Consider disabling the
+liveness probe or mTLS, or disabling mTLS for the port of the liveness probe."
+```
+See Suggested Resolutions below for an example of how to fix this by defining an
+authentication policy for your service.
+
+#### Sample 2
+
+Using the same configuration as above, which defines a liveness probe in it, but
+instead having global mTLS **disabled** and an authentication policy for the 
+service that is associated with the Pod on that port will also cause the Pod 
+to re-start and enter a CrashLoopBackOff state.
+
+An example of an authentication policy that enables mTLS for the same port as
+the liveness probe:
+
+```yaml
+apiVersion: authentication.istio.io/v1alpha1
+kind: Policy
+metadata:
+  name: httpbin
+  namespace: default
+spec:
+    targets:
+    - name: httpbin
+      ports:
+      - number: 8000
+    peers:
+    - mtls:
+```
+Again, running `kubectl get pods -l app=httpbin` would produce output
+similar to the following:
+
+```shell
+NAME                       READY     STATUS             RESTARTS   AGE
+httpbin-1a23bc456d-7ef8g   1/2       CrashLoopBackOff   3          1m
+```
+
+If the Pod is deployed in the mesh with the above configuration, the following
+note will be generated:
+
+```shell
+Summary: "mTLS and liveness probe incompatible - httpbin-1a23bc456d-7ef8g"
+
+Message: "ERROR: The pod httpbin-1a23bc456d-7ef8g in namespace default uses
+liveness probe which is incompatible with mTLS. Consider disabling the
+liveness probe or mTLS, or disabling mTLS for the port of the liveness probe."
+```
+See Suggested Resolution below for an example of how to fix this by defining an
+authentication policy for your service.
+
 
 ## Suggested Resolution
 
 You can do one of these things:
 
+- **Define an Authentication Policy for your Service.** As of Istio v0.8 you can
+  define an authentication policy that disables mTLS for the port of the 
+liveness/readiness probe.  This will override the global mTLS setting only for the
+port of the liveness/readiness probe.  The following authentication policy would
```

Comment:
It is recommended to use a port specific for health probes and disable mTLS on that port instead of disabling mTLS on the application port.